### PR TITLE
Add DashboardPresenter class and refactor dashboard controller Closes #17

### PR DESCRIPTION
**Pull request by oorion**
Add tests for confirmed and dead companies.  Update all tests to ensure most recently updated companies appear at the top of the list

### CODE REVIEW

File: app/controllers/admin/dashboard_controller.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 class Admin::DashboardController < ApplicationController
   def index
-    @uncontacted_companies = Company.where(status: "uncontacted")
-    @contacted_companies = Company.where(status: "contacted")
+    @dashboard_presenter = DashboardPresenter.new
```
Comment (laurawhalin) [inline app/controllers/admin/dashboard_controller.rb:3]: Sandi Metz would be proud!

File: app/models/dashboard_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  def confirmed_companies
+    Company.where(status: "confirmed").order(updated_at: :desc)
+  end
+
+  def dead_companies
+    Company.where(status: "dead").order(updated_at: :desc)
+  end
+end
```
Comment (laurawhalin) [inline app/models/dashboard_presenter.rb:17]: :fireworks:

File: app/views/admin/dashboard/index.html.erb [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+<div class="confirmed-companies">
+  <% @dashboard_presenter.confirmed_companies.each do |company| %>
+    <%= company.organization %>
+  <% end %>
+</div>
+<div class="dead-companies">
+  <% @dashboard_presenter.dead_companies.each do |company| %>
     <%= company.organization %>
   <% end %>
 </div>
```
Comment (laurawhalin) [inline app/views/admin/dashboard/index.html.erb:21]: Cool. The front end people can rearrange this if they want, but this gives them a clear idea of what they have to work with. Nice work!

File: spec/features/admin_dashboard_spec.rb [hunk 4]
```diff
@@ -27,6 +27,7 @@ def login
       within(".uncontacted-companies") do
         expect(page).to have_content(companies.first.organization)
         expect(page).to have_content(companies.last.organization)
+        expect(companies.first.updated_at <= companies.last.updated_at).to eq(true)
```
Comment (laurawhalin) [inline spec/features/admin_dashboard_spec.rb:30]: At first I thought this was a reverse hashrocket. Good way to check that the list appears in the right order!

Comment (cluhring) [inline spec/features/admin_dashboard_spec.rb:30]: Nice tests Orion!

**Comment by cluhring** [comment]
Reviewed - no questions.  Looks good!

**Pull request merged**